[2021-01-15 16:45:42,994] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op2 2021-01-15T07:43:50.673128+00:00 [queued]>
[2021-01-15 16:45:43,002] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op2 2021-01-15T07:43:50.673128+00:00 [queued]>
[2021-01-15 16:45:43,002] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:45:43,002] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-01-15 16:45:43,002] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:45:43,006] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): op2> on 2021-01-15T07:43:50.673128+00:00
[2021-01-15 16:45:43,008] {standard_task_runner.py:54} INFO - Started process 9518 to run task
[2021-01-15 16:45:43,058] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'noti_test', 'op2', '2021-01-15T07:43:50.673128+00:00', '--job_id', '59', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/noti_test.py', '--cfg_path', '/var/folders/4f/xnn880bn51d9945mdydbgwdw0000gn/T/tmpw24p739n']
[2021-01-15 16:45:43,060] {standard_task_runner.py:78} INFO - Job 59: Subtask op2
[2021-01-15 16:45:43,081] {logging_mixin.py:112} INFO - Running <TaskInstance: noti_test.op2 2021-01-15T07:43:50.673128+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-01-15 16:45:43,092] {logging_mixin.py:112} INFO - function2
[2021-01-15 16:45:43,093] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-01-15 16:45:43,095] {taskinstance.py:1057} INFO - Marking task as SUCCESS.dag_id=noti_test, task_id=op2, execution_date=20210115T074350, start_date=20210115T074542, end_date=20210115T074543
[2021-01-15 16:45:47,997] {local_task_job.py:102} INFO - Task exited with return code 0
